icons/mic-2.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m12 8-9.17 9.17a2.83 2.83 0 1 0 4 4L16 12" />
```

Comment:
There's an ever so tiny design guideline violation regarding the 1px padding. I could't get things to work with the SVG optimization on, it kept messing up the lines, but I got this to work, what do you think?
```suggestion
  <path d="M12.02 8 2.65 18.23a2.2 2.2 0 0 0 3.12 3.12L16 11.98" />
```

Reply:
```suggestion
  <path d="m12 8-9.04 9.06a2.82 2.82 0 1 0 3.98 3.98L16 12" />
```
Oooor, just never mind, I think I got it to work with the original design. :)

Reply:
Good point, I like the shape of the new one but I think it should be a bit more like the `edit` or `brush` icons for consistency so I made something that's in-between that also follows the rules. Should I update the icon with this one or your one?

<img width="572" alt="image" src="https://user-images.githubusercontent.com/72697755/177391389-fb967e55-6c5a-46a0-966c-3c27d751966b.png">

Reply:
I think this should be pretty much the same as my second comment?
```suggestion
  <path d="m12 8-9.04 9.06a2.82 2.82 0 1 0 3.98 3.98L16 12" />
```

![image](https://user-images.githubusercontent.com/17746067/177406396-8142491f-9da4-4faf-a802-f96f25987faf.png)